### ISSUE TITLE
What should I do for this kind of training curve?

### ISSUE THREAD

**Issue by hotelll** [Megvii-BaseDetection/YOLOX#1006]
![ap](https://user-images.githubusercontent.com/45009029/146662139-04bfbae8-d992-414c-89ee-4234b2b8abca.png)
How to improve the training? My training dataset contains 16000 images. The current training settings are:
 self.mosaic_prob = 1.0
self.mixup_prob = 1.0
self.hsv_prob = 1.0
self.flip_prob = 0.5
self.degrees = 10.0
self.translate = 0.1
self.mosaic_scale = (0.1, 2)
self.mixup_scale = (0.5, 1.5)
self.shear = 2.0
self.enable_mixup = True

# --------------  training config --------------------- #
self.warmup_epochs = 5
self.max_epoch = 150
self.warmup_lr = 0
self.basic_lr_per_img = 0.01 / 64.0
self.scheduler = "yoloxwarmcos"
self.no_aug_epochs = 15
self.min_lr_ratio = 0.05
self.ema = True
self.weight_decay = 5e-4
self.momentum = 0.9
self.print_interval = 10
self.eval_interval = 10

**Comment by Mobu59** [comment]
You can visualize your training data to check whether the data enhancement is suitable for your dataset. For small models, it is generally necessary to weaken the enhancement strength

**Comment by hotelll** [comment]
I find that the training curve becomes normal after I cancel the mixup augmentation, but the final performance drops 0.8.
